src/components/Card.tsx

```diff
@@ -23,17 +30,31 @@ const Card: React.FC<CardProps> = ({
     };
 
     return (
-        <div className="card-container" id={id}>
-            <div className="card-item">Stadium - {stadiumName}</div>
-            <div className="card-item">City - {city}</div>
-            <div className="card-item">Capacity - {capacity}</div>
-            <div className="card-item">Field - {fieldType}</div>
-            <div className="form-buttons card-button">
-                <Button type="button" onClick={handleRemove}>
+        <MiuCard id={id} className="card-container" sx={{padding: "5px"}}>
+            <CardContent sx={{padding: "4px"}}>
+                <Typography variant="body1">Stadium - {stadiumName}</Typography>
+                <Typography variant="body1">City - {city}</Typography>
+                <Typography variant="body1">Capacity - {capacity}</Typography>
+                <Typography variant="body1">Field - {fieldType}</Typography>
+            </CardContent>
+            <CardActions sx={{paddingTop: 0}}>
+                <Button
+                    sx={{
+                        backgroundColor: theme.palette.error.dark,
+                        "&:hover": {
+                            backgroundColor: theme.palette.error.main,
+                        },
+                    }}
+                    variant="contained"
+                    type="button"
+                    size="small"
+                    onClick={handleRemove}
+                    endIcon={<DeleteIcon />}
```

Comment:
What do you think about moving this button to the right? Now it`s too empty there and too much texts on the left side
![image](https://github.com/proga-react-practice/sigma-ivan-ostapiv/assets/74414048/7e3f6e10-835a-4822-a0b6-827d8bb6fb28)